src/Services/PatientTrackerService.php

```diff
@@ -167,7 +167,7 @@ public function is_tracker_encounter_exist($apptdate, $appttime, $pid, $eid)
      * @param string $enc_id
      * @return bool|int
      */
-    public function manage_tracker_status($apptdate, $appttime, $eid, $pid, $user, $status = '', $room = '', $enc_id = '')
+    public static function manage_tracker_status($apptdate, $appttime, $eid, $pid, $user, $status = '', $room = '', $enc_id = '')
```

Comment:
There is a $this used in line 273, which will likely break. Not sure how to rework that to work for a static call. @adunsulag thoughts?

Reply:
We'd need to use a singleton instance (have a static $instance property that is initialized once), or skip refactoring this method if its not used in the flowboard.